app/screens/Lists/BooksList/components/BookItem/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { View, Image, StyleSheet, Text } from 'react-native';
+
+import styles from './styles';
+
+function BookItem({ image, title, author }) {
+  return (
+    <View style={styles.itemContainer}>
+      {image ? (
```

Comment:
use a image for placeholder instead if a Image || View

Something like:
```
<Image
          style={styles.image}
          source={{ uri: image || placeholder.png <-- this imported }}
        />
```

Reply:
another option is defaultProps = {
 image: palceholder.png  <--- this imported
}

And keep the uri on image component like ´{ uri: image }`